### PR TITLE
Liq 1.2

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Review by godsflaw — APPROVED** [review]

**Review by lucas-manuel — APPROVED** [review]
LGTM

File: src/DssDeploy.t.sol [hunk 1]
```diff
@@ -205,7 +205,8 @@ contract DssDeployTest is DssDeployTestBase {
 
     function testBite() public {
         deploy();
-        this.file(address(cat), "ETH", "lump", 1 ether); // 1 unit of collateral per batch
+        this.file(address(cat), "ETH", "lump", rad(200 ether)); // 200 DAI max per batch
```
Comment (kmbarry1) [inline src/DssDeploy.t.sol]: Doesn't this need to become `dunk`? Ditto below.

Comment (kmbarry1) [reply src/DssDeploy.t.sol]: Or are we waiting on the later changes being merged to `dss` master before we do that?

Comment (godsflaw) [reply src/DssDeploy.t.sol]: Yeah, since we changed the meme, this PR will have to be updated.

**Review by godsflaw — APPROVED** [review]
LGTM

**Review by kmbarry1 — APPROVED** [review]
LGTM

**Pull request merged**